operators/src/source/gdal_source/loading_info.rs

```diff
@@ -54,6 +53,134 @@ impl MetaData<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectangle>
         Box::new(self.clone())
     }
 }
+#[derive(PartialEq, Serialize, Deserialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct GdalMetadataFixedTimes {
+    time_steps: Vec<TimeInterval>,
+    time_steps_no_gaps: Vec<TimeInterval>,
```

Comment:
why do you store both, time steps with and without gaps? Especially since you derive Deserialize it's inconvenient having to specify the time steps twice e.g. in a JSON file. I think it would be better to just have a time_steps field and require the timesteps to have no gaps and be sorted. Add a todo to check this during serialization